### PR TITLE
Add cute teaser animations to Fabric docs

### PR DESCRIPTION
**Pull request by awaelchli**
## What does this PR do?

Adds a bit of eye candy to the top of the accelerator, precision, and launch pages.
In addition, adds the "spectrum" image on the home page (donated by Adam Kuhn, thanks!)

cc @borda @carmocca @justusschock @awaelchli

### CODE REVIEW

File: docs/source-fabric/fundamentals/accelerators.rst [hunk 1]
```diff
@@ -3,6 +3,13 @@ Accelerate your code with Fabric
 ################################
 
 
+.. raw:: html
+
+    <center>
+        <video width="100%" max-width="500px" src="https://pl-public-data.s3.amazonaws.com/assets_lightning/fabric/animations/accelerators.mp4", type="video/mp4" autoplay loop muted></video>
```
Comment (awaelchli) [inline docs/source-fabric/fundamentals/accelerators.rst:9]: https://pl-public-data.s3.amazonaws.com/assets_lightning/fabric/animations/accelerators.mp4

File: docs/source-fabric/fundamentals/launch.rst [hunk 2]
```diff
@@ -15,6 +15,13 @@ To run your code distributed across many devices and many machines, you need to
 Simple Launch
 *************
 
+.. raw:: html
+
+    <center>
+        <video width="100%" max-width="500px" src="https://pl-public-data.s3.amazonaws.com/assets_lightning/fabric/animations/launch.mp4", type="video/mp4" autoplay loop muted></video>
```
Comment (awaelchli) [inline docs/source-fabric/fundamentals/launch.rst:21]: https://pl-public-data.s3.amazonaws.com/assets_lightning/fabric/animations/launch.mp4

File: docs/source-fabric/fundamentals/launch.rst [hunk 3]
```diff
@@ -49,6 +56,12 @@ You can learn more about it :ref:`here <Fabric in Notebooks>`.
 Launch with the CLI
 *******************
 
+.. raw:: html
+
+    <center>
+        <video width="100%" max-width="500px" src="https://pl-public-data.s3.amazonaws.com/assets_lightning/fabric/animations/launch-cli.mp4", type="video/mp4" autoplay loop muted></video>
```
Comment (awaelchli) [inline docs/source-fabric/fundamentals/launch.rst:62]: https://pl-public-data.s3.amazonaws.com/assets_lightning/fabric/animations/launch-cli.mp4

File: docs/source-fabric/fundamentals/precision.rst [hunk 4]
```diff
@@ -2,6 +2,11 @@
 Save memory with mixed precision
 ################################
 
+.. raw:: html
+
+    <center>
+        <video width="100%" max-width="500px" src="https://pl-public-data.s3.amazonaws.com/assets_lightning/fabric/animations/precision.mp4", type="video/mp4" autoplay loop muted></video>
```
Comment (awaelchli) [inline docs/source-fabric/fundamentals/precision.rst:8]: https://pl-public-data.s3.amazonaws.com/assets_lightning/fabric/animations/precision.mp4

File: docs/source-fabric/index.rst [hunk 5]
```diff
@@ -57,6 +57,16 @@ Fabric is the fast and lightweight way to scale PyTorch models without boilerpla
 Why Fabric?
 ***********
 
+|
+|
+
+.. figure:: https://pl-public-data.s3.amazonaws.com/assets_lightning/fabric/PyTorch-to-Fabric-Spectrum-2.svg
```
Comment (awaelchli) [inline docs/source-fabric/index.rst:63]: https://pl-public-data.s3.amazonaws.com/assets_lightning/fabric/PyTorch-to-Fabric-Spectrum-2.svg

File: docs/source-fabric/fundamentals/accelerators.rst (hunk 1 above)
Comment (carmocca) [reply docs/source-fabric/fundamentals/accelerators.rst:9]: pro tip: if you install https://github.com/refined-github/refined-github then links become clickable in the diff

**Review by carmocca — APPROVED** [review]

File: docs/source-fabric/fundamentals/accelerators.rst (hunk 1 above)
Comment (awaelchli) [reply docs/source-fabric/fundamentals/accelerators.rst:9]: Nice. I guess I still did a good job copying the links for all those who don't have the extension 😄

**Review by Borda — APPROVED** [review]

**Pull request merged**